### DISCUSSION TITLE
Bundle @livewireScripts into app.js with Vite

### DISCUSSION THREAD

**Discussion by Blinks44** [livewire/livewire#5460] [1. Help]
Greetings!
Is it possible to bundle ```@livewireScripts``` into ```app.js```, so Vite compile everything into one ```app.js``` file?
Google Lighthouse doesn't like many js includes, so i want to have one ```app.css``` and ```app.js``` file after ```vite build```.

**Comment by joshhanley** [comment]
@Blinks44 not currently possible as Livewire's scripts also has some inline scripts that is runs after the include. Maybe this might be something that is improved in V3.
